.github/ISSUE_TEMPLATE/new_node_package.yml

```diff
@@ -46,6 +46,13 @@ body:
       placeholder: ex. 0.0.0.20240513
     validations:
       required: true
+  - type: input
```

Comment:
[nitpick] I think the arguments should be placed just after `Tool's description` and before `Dependencies`. It feels for me more natural to first fill the required fields like `Category`.